pandas/_libs/tslibs/offsets.pyx

```diff
@@ -17,7 +17,7 @@ cimport numpy as cnp
 from numpy cimport int64_t
 cnp.import_array()
 
-
+cimport pandas._libs.tslibs
```

Comment:
Why?

Reply:
Cython was making noise about this when running under 2.7; looks like it doesn't raise the same error under 3. Will remove this.